api/instance/v1/server_utils.go

```diff
@@ -180,20 +178,19 @@ func (s *API) GetServerUserData(req *GetServerUserDataRequest, opts ...scw.Reque
 	return res, nil
 }
 
+// SetServerUserDataRequest is used by SetServerUserData method.
 type SetServerUserDataRequest struct {
 	Zone     utils.Zone `json:"-"`
 	ServerID string     `json:"-"`
 
-	// Key define the user data key to set
+	// Key defines the user data key to set.
 	Key string `json:"-"`
 
-	// Content define the data to set
+	// Content defines the data to set.
 	Content io.Reader
 }
 
-// SetServerUserData set user data
-//
-// Overwrites a user data with the given key on a server
+// SetServerUserData sets the content of a user data on a server with the given key.
```

Comment:
`a user data` doesn't really work since data is plural. So either `user data` or either `a user data object`.

Reply:
> SetServerUserData sets the content of user data on a server for the given key.